source/git.js

```diff
@@ -31,6 +32,8 @@ var GitExecutionTask = function(commands, repoPath, allowedCodes) {
   this.potentialError = new Error(); // caputers the stack trace here so that we can use it if the command fail later on
   this.potentialError.commmands = commands;
   this.allowedCodes = allowedCodes;
+  this.stdoutListener = undefined;
```

Comment:
This and the next line can be removed. (Should have been part of https://github.com/codingtwinky/ungit/commit/7462b586d8373ba9369505c5a0b49b3e98094f24 but we both missed it.)